### PR TITLE
[LIB-480] Dataobject increment() method

### PR DESCRIPTION
**Pull request by jakubbilko**

### CODE REVIEW

File: test/integration/dataobjectsTest.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import Syncano from '../../src/syncano';
 import {ValidationError} from '../../src/errors';
 import {suffix, credentials, createCleaner} from './utils';
 
-describe('Dataobject', function() {
+describe.only('Dataobject', function() {
```
Comment (dancio) [inline test/integration/dataobjectsTest.js]: only? :P

**Comment by dancio** [comment]
how about querySet method? there is API for that?

File: test/integration/dataobjectsTest.js (hunk 1 above)
Comment (jakubbilko) [inline test/integration/dataobjectsTest.js]: One and only :dancer:

**Comment by dancio** [comment]
lgtm

**Pull request merged**